modules/theme-tailwind/molecules/ProductParameters.vue

```diff
@@ -33,14 +33,6 @@
                 <td class="py-4 px-6 text-sm font-medium text-gray-900 whitespace-nowrap dark:text-white">
                   Apple Imac 27"
                 </td>
-                <td class="py-4 px-6 text-sm text-gray-500 whitespace-nowrap dark:text-gray-400">Stolní PC</td>
-              </tr>
```

Comment:
Why did you remove only this part when obviously the whole component is hardcoded? Why did you create new component instead of reusing this one? And even if you did, why not name the new component ProductAttributes when the module is called attributes?